apis/placement/v1/clusterresourceplacement_types.go

```diff
@@ -338,6 +338,9 @@ type ClusterSelectorTerm struct {
 	//
```

Comment:
missing tolerations hint

Reply:
Hi Zhiying! The hint for the tolerations is added in the CRP field part (line 164) rather than the struct comment part (line 667) as the struct comment is not incorporated into the CRD.